protos/schedules.proto

```diff
@@ -0,0 +1,29 @@
+syntax="proto3";
+
+package dataform.schedules;
+
+message Notification {
+  repeated string emails = 1;
+  bool success = 2;
```

Comment:
these are going to appear in JSON form in a checked in project somewhere in the end, so suggest:
`on_success` and `on_failure`